### DISCUSSION TITLE
Basic tag search is ready!

### DISCUSSION THREAD

**Discussion by renerocksai** [nvim-telekasten/telekasten.nvim#35] [Announcements]
To make it quick: We have a new function / Telekasten command `show_tags` and the config option `show_tags_theme`.

`show_tags()` brings up a tag picker like this:

![image](https://user-images.githubusercontent.com/30892199/145666175-b581b273-7c0f-4361-b12f-f44b169d4684.png)

Supports `<c-i>` (insert tag), `<c-y>` (yank tag) and `<cr>` (search notes for selected tag)

You can bind it, like this:

```vim
nnoremap <leader>za :lua require('telekasten').show_tags()<CR>
nnoremap <leader># :lua require('telekasten').show_tags()<CR>
inoremap <leader># <cmd>lua require('telekasten').show_tags({i = true})<cr>
```

**Comment by lokesh-krishna** [comment]
Is it possible to use Telescope for the tag picker?

**Reply by renerocksai** [reply]
I don't understand. The tag picker is a telescope picker.

**Reply by lokesh-krishna** [reply]
Oh, my bad then. It seemed to be styled differently which confused me. Didn't seem to be using the style that's used for finding notes so I thought this was something else.

**Reply by renerocksai** [reply]
There are 3 'themes' to choose from. Since tags aren't files, a preview pane doesn't make sense. Hence the popup / bottom panel style.
